python/packages/ai/teams/ai/ai_history_options.py

```diff
@@ -4,12 +4,9 @@
 """
 
 from dataclasses import dataclass
-from enum import Enum
+from typing import Literal
 
-
-class AssistantHistoryType(str, Enum):
-    TEXT = "__TEXT__"
-    PLAN_OBJECT = "__PLAN_OBJECT__"
+AssistantHistoryType = Literal["text", "planObject"]
```

Comment:
Can you share why we prefer this over an Enum?

Reply:
String literals give you intellisense in the editor as you are working and do not require you to import a class/enum:
![image](https://github.com/microsoft/teams-ai/assets/44811138/0c7d13c1-a52c-4552-97a1-9cabd61227d4)

Reply:
Do we need to align all Enums to Literal? How do we decide when to use Literal and when to use Enum?

Reply:
My _personal opinion_ is that literals should be used for public types so that our users get intellisense and doesn't have to import another class from the library, thus making boilerplate smaller and easier to read. I am also for using literals everywhere in the codebase for this same reason, so if you think it makes sense to change all our enums to literal string types I support that and I can add it as a cleanup work item.